rules/cloud/azure_network_p2s_vpn_modified_or_deleted.yml

```diff
@@ -0,0 +1,22 @@
+title: Azure Point-to-site VPN Modified or Deleted
+id: d9557b75-267b-4b43-922f-a775e2d1f792
+description: Identifies when a Point-to-site VPN is Modified or Deleted.
+author: Austin Songer
+status: experimental
+date: 2021/08/08
+references:
+    - https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations
+logsource:
+  service: azure.activitylogs
+detection:
+    selection:
+        properties.message:
+            - MICROSOFT.NETWORK/P2SVPNGATEWAYS/WRITE
+            - MICROSOFT.NETWORK/P2SVPNGATEWAYS/DELETE
```

Comment:
Not this ?
- microsoft.network/p2sVpnGateways/reset/action
- Microsoft.Network/p2sVpnGateways/generatevpnprofile/action
- Microsoft.Network/p2sVpnGateways/disconnectp2svpnconnections/action
- Microsoft.Network/p2sVpnGateways/providers/Microsoft.Insights/diagnosticSettings/write